custom/enikshay/tasks.py

```diff
@@ -193,9 +193,9 @@ def get_adherence_schedule_start_date(self):
         if not raw_date:
             return None
         if parse_datetime(raw_date):
-            return parse_datetime(raw_date)
+            return (parse_datetime(raw_date)).replace(tzinfo=pytz.utc)
```

Comment:
can you explain what outcome you are expecting here? I'm a bit confused by what is happening. I think `raw_date` is actually potentially in IST as it is coming from the phone? I don't think that is encoded in the date itself, so you might need to first tell python that the TZ of the date is IST before converting it to UTC. 

I believe the dates we've been using in this project are in IST, since that is what gets saved to the case. (for [example](https://github.com/dimagi/commcare-hq/blob/7e86b09be40e9bd642dd97a495a49d940aba2dcc/custom/enikshay/integrations/ninetyninedots/utils.py#L88-L97))